model/actors/miner/sector.go

```diff
@@ -113,7 +217,7 @@ func (ml MinerSectorInfoList) Persist(ctx context.Context, s model.StorageBatch,
 		return nil
 	}
 
-	if version.Major != 1 {
+	if version.Major == 1 {
```

Comment:
Should be `!=` but as above this is currently irrelevant since there is only one major version of the schema currently: v1

Reply:
Assuming we'll add a major version 2 someday wouldn't this be better as ` == 0` (what I thought I typed when I pushed this)